applications/nrf_desktop/doc/usb_state.rst

```diff
@@ -205,3 +209,22 @@ The |usb_state| is a transport for :ref:`nrf_desktop_config_channel` when the ch
 
 The module also handles a HID keyboard LED output report received through USB from the connected host.
 The module sends the report using :c:struct:`hid_report_event`, that is handled either by :ref:`nrf_desktop_hid_state` (for peripheral) or by the :ref:`nrf_desktop_hid_forward` (for dongle).
+
+nRF54H20 support
+================
+
+Due to the nRF54H20 USB device controller (UDC) characteristics, following changes have been made in the USB state module to support the nRF54H20 platform:
+
+* The USB state module spawns a separate thread to handle initializing, enabling, and disabling the USB stack.
+* The module disables the USB stack when the USB cable is disconnected and enables the stack when the cable is connected.
+
+Those changes are only applied to the nRF54H20 platform and do not affect other platforms.
+They are necessary to ensure proper USB stack operation on the nRF54H20 platform.
+
+The USB stack cannot be initialized from the System workqueue thread, because it causes a deadlock.
+Because of that, the separate thread is used to handle USB stack initialization.
+For more details, see the :ref:`CONFIG_DESKTOP_USB_INIT_THREAD <config_desktop_app_options>` Kconfig option.
+The UDC is powered down whenever the USB cable is disconnected and doesn't call appropriate callbacks to the USB stack.
+It may lead to the USB stack ending in a broken state.
+Disabling the USB stack on cable disconnection workarounds this issue.
```

Comment:
```suggestion
You can workaround this issue by disabling the USB stack upon disconnecting the cable.
```

Reply:
Maybe `The USB stack is disabled upon disconnecting the cable to work around this issue.`?

Reply:
thanks for the explanation - sounds good!